pkg/apis/servicecatalog/validation/serviceclass.go

```diff
@@ -73,17 +75,26 @@ func ValidateClusterServiceClass(serviceclass *sc.ClusterServiceClass) field.Err
 	return internalValidateClusterServiceClass(serviceclass)
 }
 
-func internalValidateClusterServiceClass(serviceclass *sc.ClusterServiceClass) field.ErrorList {
+// ValidateClusterServiceClassUpdate checks that when changing from an older
+// ClusterServiceClass to a newer ClusterServiceClass is okay.
+func ValidateClusterServiceClassUpdate(new *sc.ClusterServiceClass, old *sc.ClusterServiceClass) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, internalValidateClusterServiceClass(new)...)
+
+	return allErrs
+}
+
+func internalValidateClusterServiceClass(clusterserviceclass *sc.ClusterServiceClass) field.ErrorList {
```

Comment:
nit: I think the go way would be `func validateClusterServiceClass`

Reply:
+1, inherited this from the old code but I think you're right; it's probably a good time to change it. I didn't see this in the broker validations, so it's worth being consistent.